torch/fx/subgraph_rewriter.py

```diff
@@ -309,7 +309,10 @@ def get_replacement_nodes(curr_node: Node):
             replacement_nodes.append(curr_node)
 
         for ret_node in copied_returning_nodes:
-            get_replacement_nodes(ret_node)
+            if ret_node in match.placeholder_nodes:
+                replacement_nodes.append(ret_node)
```

Comment:
I think this should be an empty list, since there are no nodes that are created in the replaced graph.